projects/rero/ng-core/src/lib/record/editor/type/textarea/textarea.component.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnInit
+} from '@angular/core';
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import { FormlyFieldTextArea } from '@ngx-formly/bootstrap';
+
+@Component({
+  selector: 'ng-core-editor-formly-field-textarea',
+  template: `
+    <textarea
+      [formControl]="formControl"
+      [cols]="to.cols"
+      [rows]="to.rows"
+      class="form-control"
+      [class.is-invalid]="showError"
+      [formlyAttributes]="field"
+    >
+    >
+    </textarea
+    >
+    <ng-container
+      *ngIf="
+        field.templateOptions.limitWords || field.templateOptions.displayWords
+      "
+    >
```

Comment:
Why not one a single line ? (as it used ini all other component (when possible))